ocean_lib/ocean/util.py

```diff
@@ -75,18 +73,15 @@ def get_web3_connection_provider(
         raise AssertionError(msg)
 
 
-def get_contracts_addresses(address_file: str, network: str) -> Dict[str, str]:
-    return get_contracts_addresses_web3(network, address_file)
+def get_contracts_addresses(config) -> Dict[str, str]:
+    return get_contracts_addresses_web3(config)
```

Comment:
this is an older thing actually, we need the function in two modules and we always had trouble with the imports. I can look into moving this

Reply:
OK. I just created a github issue to handle it. I assigned it to you since it seems you know exactly what you want:)

https://github.com/oceanprotocol/ocean.py/issues/970